README.md

```diff
@@ -416,9 +416,31 @@ There are just a few small changes we have to make before our WebSocket applicat
 
 ### WebJars Locator
 
-Due to a [known issue](https://github.com/webjars/webjars-locator/issues/78) with the `webjars-locator` dependency and its compatibility with WebSphere (both traditional and Liberty), it doesn't work properly without some modifications (which are proposed in the linked issue). At the moment, we do not provide a modified version of this artifact which is WebSphere compatible; you'll have to modify the dependency URLs in `/src/main/resources/static/index.html` to include the version numbers of the dependencies or else they won't be found by the resource handler.
+Because Liberty uses the `wsjar` prefix by default instead of the standard `jar` prefix, `webjar-locator` won't be compatible with Liberty without some additional configuration. To resolve this issue, make the following changes:
 
-For example, you'll need to change `/webjars/bootstrap/css/bootstrap.min.css` to `/webjars/bootstrap/3.3.7/css/bootstrap.min.css` and so on. The version numbers of each dependency are listed in `pom.xml`. 
+#### Update Server Configuration
+
+Add the following line to `server.xml`:
+
+```
+<classloading useJarUrls="true" />
+```
+
+According to our [documentation](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.base.doc/ae/rwlp_config_classloading.html), this flag determines whether to use `jar:` or `wsjar:` URLs for referencing files in archives. This will allow `webjars-locator` to find the JARs it needs. 
+
+#### Update Resource Handler
+
+Recall that we added a resource handler in `WebConfig.java` in order to map the static resources on the classpath to public URLs. Replace that line with the following code:
```

Comment:
Since WebConfig.java is new in its entirety, why not just provide the complete solution including this code change in one place rather than providing the code and then updating it?